### ISSUE TITLE
SyntaxError: Unexpected token 'export'

### ISSUE THREAD

**Issue by jvdeijnden** [nuxt/typescript#384]
When building for production and trying to run the development server, this happens:

```
FATAL  Unexpected token 'export'

  export default {
  ^^^^^^

  SyntaxError: Unexpected token 'export'
  at Object.compileFunction (vm.js:341:18)
  at Generator.next (<anonymous>)


   ╭────────────────────────────────────────────╮
   │                                            │
   │   ✖ Nuxt Fatal Error                       │
   │                                            │
   │   SyntaxError: Unexpected token 'export'   │
   │                                            │
   ╰────────────────────────────────────────────╯

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! test@1.0.0 dev: `nuxt`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the test@1.0.0 dev script.
```

**Project settings**
? Project name: test
? Programming language: TypeScript
? Package manager: Npm
? UI framework: Buefy
? Nuxt.js modules: Axios, Progressive Web App (PWA), Content
? Linting tools: ESLint
? Testing framework: Jest
? Rendering mode: Universal (SSR / SSG)
? Deployment target: Server (Node.js hosting)
? Development tools: jsconfig.json (Recommended for VS Code)

**To reproduce**
run `npm run build`
run `npm run dev`

**Expected behavior**
I'm expecting the same thing to happen with a JavaScript project, which is compiling for and running the development server.

I have tried deleting the dist directory, but that doesn't fix it.

**Comment by kevinmarrec** [comment]
@jvdeijnden It is not reproducible, I've setup a project with exactly same options you selected, and `npm run build` & `npm run dev` work both fine.

If you find a reproduction, please submit it around the https://github.com/nuxt/create-nuxt-app repository.

**Comment by indus** [comment]
I'm facing the same issue. I would guess it is related to TypeScript.

**Comment by daniluk4000** [comment]
Same issue with nuxt-ts only, nuxt works fine

**Comment by kevinmarrec** [comment]
It should have been fixed with `create-nuxt-app@3.2.0`, following https://github.com/nuxt/create-nuxt-app/pull/568 which fixed an issue regarding wrong `package.json` `dev` script.
